ee/session_recordings/session_summary/local/input_data.py

```diff
@@ -31,17 +31,18 @@ def _ch_client_local_reads_prod():
 def _get_production_session_metadata_locally(
```

Comment:
That's the method to work/summarize production sessions when working locally (as testing on local sessions is miserable).